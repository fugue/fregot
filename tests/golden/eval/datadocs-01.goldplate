{
    "command": "fregot",
    "arguments": [
        "eval", "data", "dir1"
    ],
    "asserts": [
        {"exit_code": 0},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ]
}
