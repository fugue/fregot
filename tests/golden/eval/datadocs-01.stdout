[{"dir2":{"foo":"bar"}}]
